src/link.js

```diff
@@ -40,7 +44,7 @@ export const Link = BaseLink.extend({
       openOnClick: false,
```

Comment:
I'd like us to use a positive instead of a negative option name; double negation still confuses me :-)

Adding it to the list of options also allows us to set `true` as default.

(`enableOpenInNewWindow` would also be fine, but it's much longer, and what we're enabling is setting the `target` attribute)

```suggestion
      openOnClick: false,
      enableTarget: true,
```